core/src/main/java/org/mskcc/cbio/portal/scripts/UpdateGenePanel.java

```diff
@@ -55,25 +55,22 @@ public void run() {
             String description = "Update gene panel files.";
 
             OptionParser parser = new OptionParser();
-            OptionSpec<String> data = parser.accepts( "data",
```

Comment:
I have the auto formatter setting on in VS code, hence why so many lines are changed